Modules/AddonCompartment/AddonCompartmentModule.lua

```diff
@@ -0,0 +1,16 @@
+BattlePetCompletionist = LibStub("AceAddon-3.0"):GetAddon("BattlePetCompletionist")
```

Comment:
Please add the license header above (just copy it from one of the other lua files)

Reply:
Added.